src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,23 @@
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search. In this case, you used a one-letter variable 'e' which may be confusing. It's better to use a more descriptive name, like 'event' or 'changeEvent'.